src/style-spec/CHANGELOG.md

```diff
@@ -1,23 +1,29 @@
-# 13.23.1
+## 13.24.0
```

Comment:
We published the beta as 13.24.0 right? I'm just thinking this technically could have been a patch release, but that ship may have sailed.

Reply:
Yes, we decided to go with a full release for the main project, and I've been assuming that the same applied to the style spec.

Reply:
It's not a big deal, but I'd say the style-spec doesn't have to follow the release structure of the main module.